etc/releasing.md

```diff
@@ -230,6 +230,10 @@ The following credentials are required. Ask for these from a team member if nece
   GRS_CONFIG_USER1_USERNAME=<username>
   GRS_CONFIG_USER1_PASSWORD=<password>
   ```
+- Snyk credentials. Save these to `~/.secrets/snyk-creds.txt`:
+  ```bash
+  SNYK_API_TOKEN=<token>
+  ```
```

Comment:
Suggest adding the `SILK_CLIENT_ID` and `SILK_CLIENT_SECRET` needed for "Update Silk" steps below.

Reply:
Done.